src/include/str_list.h

```diff
@@ -7,22 +7,103 @@
 #include <string>
 #include <vector>
 
+
+/**
+ * Parse **str* and set **str_list** using these delimiters : ";,= \t"
+ * 
+ * While **pos** is inferior to the size of **str**,
```

Comment:
It is slightly confusing to refer to variables internal to the function. This paragraph seems unnecessary.